ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -569,6 +576,14 @@ def setUp(self):
         self.log("From upper layer number of packets: %d" % self.nr_vl_pkts)
         self.log("VMs: %s" % str(self.test_params['arista_vms']))
 
+        self.log("Reboot type is %s" % self.reboot_type)
+
+        if self.reboot_type == 'warm-reboot':
+            # Pre-generate list of packets to be sent in send_in_background method.
+            generate_start = datetime.datetime.now()
+            self.generate_bidirectional()
+            self.log("%s packets are ready after: %s" % (len(self.packets_list), str(datetime.datetime.now() - generate_start)))
```

Comment:
Probably it's better to use %d, instead of the first %s.

Reply:
Imroved few more loggings.